docs.html

```diff
@@ -129,9 +121,7 @@ <h4>Demo</h4>
 
           <section id="avatar">
             <h1>Avatar</h1>
-            <p>
-              Avatars are used to represent people, companies, products, etc.
-            </p>
+            <p>Avatars are used to represent people, companies, products, etc.</p>
             <h4>Demo</h4>
             <div class="well">
```

Comment:
What does the well class do?

Reply:
It adds a border around content to make it stand out. I saw this used in bootstrap. 

I have been meaning to change the name. Will do it in next refactor